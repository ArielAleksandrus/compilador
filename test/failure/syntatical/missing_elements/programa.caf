int func(){
	retorne 1;
}