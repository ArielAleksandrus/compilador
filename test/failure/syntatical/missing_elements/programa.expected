ERROR => Line 3, col 1: Expected block 'programa'
